testsuite/features/support/constants.rb

```diff
@@ -1341,6 +1341,13 @@
         sl-micro-6.1-pool-x86_64
         sl-micro-6.1-devel-uyuni-client-x86_64
       ],
+    'tumbleweed' =>
+      %w[
+        opensuse_tumbleweed-x86_64
+        opensuse_tumbleweed-updates-x86_64
+        opensuse_tumbleweed-uyuni-client-x86_64
+        opensuse_tumbleweed-uyuni-client-devel-x86_64
+      ],
```

Comment:
I don't think we need that one.

When we sync Tumbleweed, we do partially, using this:

```
When I use spacewalk-common-channel to add channel "opensuse_tumbleweed" with arch "x86_64"
And I kill running spacewalk-repo-sync for "opensuse_tumbleweed-x86_64" channel
And I use spacewalk-repo-sync to sync channel "opensuse_tumbleweed-x86_64" including only client tools dependencies
And I use spacewalk-common-channel to add all "tumbleweed-client-tools-x86_64" channels with arch "x86_64"
```

Reply:
https://ci.suse.de/view/Manager/view/Uyuni/job/uyuni-master-infra-reference-NUE/1509/console

```
11:35:35  When I use spacewalk-common-channel to add all "tumbleweed" channels with arch "x86_64" # features/step_definitions/command_steps.rb:167
11:35:35        Synchronization error, channel tumbleweed or tumbleweed-x86_64 in Uyuni product not found (ScriptError)
11:35:35        ./features/step_definitions/command_steps.rb:171:in `/^I use spacewalk-common-channel to add all "([^"]*)" channels with arch "([^"]*)"$/'
11:35:35        features/reposync/reference_srv_sync_products_extra.feature:16:in `I use spacewalk-common-channel to add all "tumbleweed" channels with arch "x86_64"'
```

Reply:
This is for the reference server, not for tests. In the reference server we have full mirrored repos